compiler/rustc_passes/src/stability.rs

```diff
@@ -632,6 +634,7 @@ impl<'tcx> Visitor<'tcx> for MissingStabilityAnnotations<'tcx> {
         let impl_def_id = self.tcx.hir().local_def_id(self.tcx.hir().get_parent_item(ii.hir_id()));
         if self.tcx.impl_trait_ref(impl_def_id).is_none() {
             self.check_missing_stability(ii.def_id, ii.span);
+            self.check_missing_const_stability(ii.def_id, ii.span);
```

Comment:
The other call site checks `staged_api` and `is_const` before even calling `check_missing_const_stability`, seems better to make this consistent?

Reply:
Which would you prefer? I just kept things consistent with how they were already, though I definitely see the potential for confusion. The change should be made for both `check_missing_stability` and `check_missing_const_stability` either way I would think.

Reply:
Probably checking things inside check_missing_const_stability makes more sense, since it avoids code duplication.